quality/git-secrets/patterns

```diff
@@ -0,0 +1,2 @@
+password\s*=\s*.+
+.*[a-zA-Z0-9][0-9]{1,}
```

Comment:
Is there a way for apps to update or add patterns?

Reply:
deleted this file since we migrated to Gitleaks and (aligned with iOS) to use Gitleaks default config.

Reply:
@leila-gu 
But, is there a way to update the default config?

Reply:
gitleaks --config <path to config file> can be set to create a default file in the root project, and can be edited to add or allow a list of secret detection rules.
Does it make sense to create an empty config and personalize it according to the finding of each project?

Reply:
My point is more on the guideline part.
We can mention how to configure the rules in the readme.

Reply:
Sure, I'll add sort guidelines and the link to Gitleaks config documentation.

Reply:
Hi @vincentdelmundo 
I added the documentation.